OsmAnd/res/values/strings.xml

```diff
@@ -58,37 +58,37 @@
     <string name="weather_wind_kilimeters_per_hour">Kilometers per hour</string>
     <string name="weather_wind_meters_per_second">Meters/second</string>
     <string name="weather_widgets_secondary_desc">Measurement units can be changed in the Weather plugin settings.</string>
-    <string name="air_pressure_widget_desc">Displays the air pressure for the current map center.</string>
-    <string name="clouds_widget_desc">Shows the cloud coverage for the current map center.</string>
-    <string name="wind_widget_desc">Displays the wind speed for the current map center.</string>
-    <string name="precipitation_widget_desc">Shows the precipitation for the current map center.</string>
-    <string name="temperature_widget_desc">Shows the temperature for the current map center.</string>
-    <string name="weather_widget_group_desc">A set of widgets showing various weather information.</string>
+    <string name="air_pressure_widget_desc">Shows air pressure for current map center.</string>
+    <string name="clouds_widget_desc">Shows clouds for current map center.</string>
+    <string name="wind_widget_desc">Shows wind speed for current map center.</string>
+    <string name="precipitation_widget_desc">Shows precipitation for current map center.</string>
+    <string name="temperature_widget_desc">Shows temperature for current map center.</string>
+    <string name="weather_widget_group_desc">A set of widgets showing various weather info.</string>
     <string name="weather_pressure_hpa">hPa</string>
     <string name="weather_pressure_inhg">inHg</string>
     <string name="weather_pressure_mmhg">mmHg</string>
     <string name="weather_pressure_hectopascals">Hectopascals</string>
     <string name="weather_pressure_inches_of_mercury">Inches of mercury</string>
     <string name="weather_pressure_millimeters_of_mercury">Millimeters of mercury</string>
-    <string name="weather_prefs_descr">24h hour &amp; 7 day forecast, updated every 3 hours</string>
+    <string name="weather_prefs_descr">Daily and 7-day forecast updated every 3 hours.</string>
```

Comment:
Wrong meaning: "Daily forecast" would mean it's updated once per day, would contradict the "every 3 hours".

Reply:
I see the point.